javascript/javascript1/week1/flight_booking.js

```
@@ -0,0 +1,52 @@
+function getFullname(firstname, surname, useFormalName = false){   // I added false as the default boolean value, if we don't have false or true as an argument
+    if(typeof(firstname) === "string" && typeof(surname) === "string"){
+        if(useFormalName){
+            return ("Lord " + firstname + " " + surname);
+        } else{
+            return (firstname + " " + surname);
+        }
+    } else{
+        return ("Firstname or surname are not correct!");
```

On this line, instead of returning a string, you could use `throw` expression. [You can read more about it here.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw)
This is also might be bit advanced for now, but you can try to replace this line with `throw "Firstname or surname are not correct!"` ans see in the console what it says there.